repository-s3/src/main/resources/plugin-descriptor.properties

```
@@ -0,0 +1,39 @@
+# Elasticsearch plugin descriptor file
+# This file must exist as 'plugin-descriptor.properties' inside a plugin.
+#
+### example plugin for "foo"
+#
+# foo.zip <-- zip file for the plugin, with this structure:
+# |____   <arbitrary name1>.jar <-- classes, resources, dependencies
+# |____   <arbitrary nameN>.jar <-- any number of jars
+# |____   plugin-descriptor.properties <-- example contents below:
+#
+# classname=foo.bar.BazPlugin
+# description=My cool plugin
+# version=6.0
+# elasticsearch.version=6.0
+# java.version=1.8
+#
+### mandatory elements for all plugins:
+#
+# 'description': simple summary of the plugin
+description=Aiven AWS S3 Cloud Storage Repository Plugin
```

`Cloud Storage` is seemingly a remnant from `Google Cloud Storage`
```suggestion
description=Aiven AWS S3 Repository Plugin
```